Samples/Common/Sources/LibraryBridge/RecordingSample.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RecordingSample.swift
+//
+//  Created by Nikolay Volosatov on 2024-06-23.
+//
+//  Copyright (c) 2012 Karl Stenerud. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall remain in place
+// in this source code.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+import Foundation
+import KSCrashRecording
+
+public class RecordingSample {
+    public static func simpleInstall() {
+        let config = KSCrashConfiguration()
```

Comment:
What do you think? Does this name look ok in swift?

Reply:
Not sure. I would want to have just `install` if I don't care about config. 

And also maybe having a way to inline config creation like:
```swift
KSCrash.shared().install(with: .init(monitors: [.asyncSafe]))
```
But this last idea is the one I'm not sure about here.